docs/FrontierDevelopments-CAPI-endpoints.md

```diff
@@ -466,12 +531,18 @@ provides access to information about the Cmdr's fleet carrier.
         1. `buyPrice`: Price to buy commodity FROM carrier
         1. `sellPrice`: Price to sell commodity TO carrier
         1. `demand`: Number of commodity carrier is willing to buy
+        1. `legality`: ?? 
+        1. `meanPrice`: ??
+        1. `demandBracket`: ??
         1. `stockBracket`: ??
         1. `locName`: Localized name of commodity, depending on fleet carrier owner's locale
+
 1. `ships`: Shipyard information
    * `shipyard_list`: List containing available ships
+
 1. `modules`: Available modules in outfitting
 
+1. `timestamp`: Timestamp of last update in ISO 8601 interchange standard, using `Z` as the UTC timezone suffix.
```

Comment:
I can't find this in current Live ghalaxy `/fleetcarrier` output.  Is it being added by whatever you're using to query ?

Reply:
Mmm, I grabbed this directly from the `CAPIData.data` being passed from the new EDMC code. I can now see that the timestamp is injected into the `CAPIData` object in `companion.py` so that's where it came from.   Will remove.

Reply:
I wondered if that was the case :) .